### DISCUSSION TITLE
date picker/calendar

### DISCUSSION THREAD

**Discussion by zaamaulan** [cosscom/coss#168] [Suggestions]
calendar, date picker, date time picker as in this design
https://craft.mxkaske.dev/demo/data-table?date=1734627600000-1734713999999

**Comment by itsbrunocampos** [comment]
Hey @zaamaulan, they have just released a brunch of date picker components with some variations that are pretty close to what you need and might be easy to adjust further if needed. Check the new calendar components out on their website:
https://originui.com/calendars-date-pickers
